2016/macros.ddoc

```diff
@@ -79,7 +79,7 @@ $(DIV class="page_right equal",
   )
   $(DIV class="twitter",
     $(DIV class="hash", )
-<a class="twitter-timeline" href="https://twitter.com/search?q=%23dconf+%23dlang" data-widget-id="405059685734023169">Tweets about "#dconf #dlang"</a>
+<a class="twitter-timeline"  href="https://twitter.com/search?q=%23dconf%20%23dlang%20-RT" data-widget-id="726813389926883328">Tweets about "#dconf #dlang"</a>
```

Comment:
You added an extra space now, before `href`. Or does Twitter give it out like that?